common/test/db_fixtures/waffle_flags.json

```diff
@@ -0,0 +1,18 @@
+[
+    {
+        "pk": 1,
+        "model": "waffle.flag",
+        "fields": {
+            "name": "course_experience.course_outline_page",
```

Comment:
Note to Reviewers: This flag was originally in common/test/db_cache/bok_choy_data_default.json where it never should have lived.  Long story how it got there.  This PR removes it from the json file and moves it here.